domains/oogway.json

```diff
@@ -1,9 +1,8 @@
 {
     "owner": {
-        "username": "ActualMasterOogway",
-        "email": "williasfun@gmail.com"
+        "username": "ActualMasterOogway"
```

Comment:
you cant just have your username there. if you dont want to use your email leave it blank and add in there your discord, twitter etc.